com.lettier.movie-monad.json

```diff
@@ -0,0 +1,1353 @@
+{
+  "app-id": "com.lettier.movie-monad",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "writable-sdk": true,
+  "command": "movie-monad",
+  "rename-desktop-file": "movie-monad.desktop",
+  "rename-icon": "movie-monad-icon",
+  "rename-appdata-file": "movie-monad.appdata.xml",
```

Comment:
Would be nice for you to rename these upstream.

Reply:
This isn't required as the Flatpak system has built-in accommodations for such a situation. 

:+1:

Reply:
Those options are workarounds not solutions. Users will see two copies and two separate reviews for example because your appstream ids are different.

The domain style ids are recommended by the upstream desktop and appstream specs so they are the best option going forward.